critiquebrainz/data/model/user.py

```diff
@@ -57,31 +63,23 @@ def list(cls, limit=None, offset=None):
         return users, count
 
     def has_voted(self, review):
-        if self._votes.filter_by(revision=review.last_revision).count() > 0:
-            return True
-        else:
-            return False
+        return self._votes.filter_by(revision=review.last_revision).count() > 0
 
     @property
     def is_review_limit_exceeded(self):
-        if self.reviews_today_count() >= self.user_type.reviews_per_day:
-            return True
-        else:
-            return False
+        return self.reviews_today_count() >= self.user_type.reviews_per_day
 
     @property
     def is_vote_limit_exceeded(self):
-        if self.votes_today_count() >= self.user_type.votes_per_day:
-            return True
-        else:
-            return False
+        return self.votes_today_count() >= self.user_type.votes_per_day
```

Comment:
I can accept changes to 3 methods above, the rest doesn't have much point since we don't exactly follow PEP 8. For example, we don't use maximum 79 character length rule.